### ISSUE TITLE
Organization about component

### ISSUE THREAD

**Issue by andrewtavis** [activist-org/activist#129]
### Terms

- [X] I have searched [open and closed feature requests](https://github.com/activist-org/activist/issues?q=is%3Aissue+label%3Afeature)
- [X] I agree to follow activist's [Code of Conduct](https://github.com/activist-org/activist/blob/main/.github/CODE_OF_CONDUCT.md)

### Description

This issue is to create parts of the about component that is being made in https://github.com/activist-org/activist/issues/127. We want the features made in this issue to be conditionally switched to based on the whether `organization` is passed to the aforementioned component's props :)

The designs for this component can be found on the [Components tab in Figma](https://www.figma.com/file/I9McFfaLu1RiiWp5IP3YjE/activist_public_designs?type=design&node-id=1606%3A585&t=5CR5mIoy4P0OjwjS-1) and can further be seen below:

![Org Information](https://github.com/activist-org/activist/assets/24387426/8d05795e-093b-4ca2-a511-03fc9092444b)

Mobile:

![About](https://github.com/activist-org/activist/assets/24387426/507e2d10-d060-4743-8aa4-3afbfb381556)

- This issue is blocked by https://github.com/activist-org/activist/issues/127 that makes the baseline component.

### Contribution

Will be working on this with people in the coming days :) Happy to support or work on it myself 😊

**Comment by andrewtavis** [comment]
Note that this component and the one in #128 need to have the same height for consistency.

**Comment by andrewtavis** [comment]
Assigning myself to hold this issue for the team working on it for Code Night 🌙😊

**Comment by momanyisamuel** [comment]
Hi @andrewtavis gladly work on this as well

**Comment by andrewtavis** [comment]
762a874 finalized this and #128, @momanyisamuel :) I'll continue the work on this in #198 for the finalized versions at various breakpoints :)

Btw, as of now the `line-clamp-1` CSS isn't working, but by the looks of it we just need to update the version of `nuxtjs/tailwindcss`. We likely don't need `tailwindcss` in the dependencies by itself either :) You'd be welcome to look into these updates and getting the clamp text to work properly with just the Nuxt Tailwind dependency in another issue.
